source/concepts/faq.md

```diff
@@ -59,3 +59,16 @@ There are downsides to relying on [experimental features](https://nixos.org/manu
   Improvements to experimental features have a low priority.
 - The [Nix documentation team](https://nixos.org/community/teams/documentation.html) focuses on improving documentation and learning materials for stable features and common principles.
   When using flakes, you will have to rely more heavily on user-to-user support, third-party documentation, and the source code.
+
+## Are there any known impurities in builds?
+
+Yes, there are:
+
+- CPU architecture—great effort being made to avoid compilation of native instructions in favour of hardcoded supported ones.
+- System's current time/date.
+- Linux kernel parameters e.g. [IPv6 capabilities](https://github.com/NixOS/nix/issues/5615).
+- Timing behaviour of the build system—parallel Make build does not get the correct inputs in some cases.
+- Builds that access the home directory or fetch data from the Internet.
```

Comment:
The home directory can't be accessed in the sandbox, so that shouldn't be listed here.

Reply:
I understand. However, since sandboxing is not enabled by default, wouldn't it be pertinent to add it to the list? How about:

```suggestion
- Builds with sandboxing disabled that access the home directory or fetch data from the Internet.
```

Reply:
It is enabled by default on Linux, though disabled by default on Darwin. But once the sandbox is disabled you can do pretty much anything and it doesn't make sense to have a list for that.

I do think it's worthwhile to mention at the beginning that these are impurities left when the sandbox is enabled.